src/lib/style/components/_query-filter.scss

```diff
@@ -1,10 +1,11 @@
 @use '../abstracts/variables' as SRC;
 
 .QueryFilter {
-  margin: 0rem 1rem 1rem 0;
-  border: 1px solid SRC.$border-color-gray;
-  padding: 1rem;
-
+  &__title {
+    font-weight: bold;
+  }
+  padding: 2rem;
+  background-color: rgba(47, 142, 148, 0.05);
```

Comment:
Do you know if you are supposed to use the lighten version of the theme color here?

Reply:
Nice catch, I'll look into that.

Reply:
Updated!